pcdsdevices/fms.py

```diff
@@ -0,0 +1,259 @@
+"""
+CXI:SETRA:01:PERIODS_RBV 60 monitor
+CXI:SETRA:01:DELAY_TIME_RBV 60 monitor
+CXI:SETRA:01:SAMPLE_TIME_RBV 60 monitor
+CXI:SETRA:01:HOLD_TIME_RBV 60 monitor
+CXI:SETRA:01:SAMPLE_MODE_RBV 60 monitor
+CXI:SETRA:01:MASS_MODE_RBV 60 monitor
+CXI:SETRA:01:RECIPE_MODE_RBV 60 monitor
+CXI:SETRA:01:RECORD_REQ 60 monitor
+CXI:SETRA:01:RH 60 monitor
+CXI:SETRA:01:BP 60 monitor
+CXI:SETRA:01:TEMP 60 monitor
+CXI:SETRA:01:DATE 60 monitor
+CXI:SETRA:01:TIME 60 monitor
+CXI:SETRA:01:TVOC 60 monitor
+CXI:SETRA:01:CO2 60 monitor
+CXI:SETRA:01:FLOW_RATE 60 monitor
+CXI:SETRA:01:SAMPLE_DURATION 60 monitor
+CXI:SETRA:01:CH0_SIZE 60 monitor
+CXI:SETRA:01:CH0_MASS 60 monitor
+CXI:SETRA:01:CH1_SIZE 60 monitor
+CXI:SETRA:01:CH1_MASS 60 monitor
+CXI:SETRA:01:CH2_SIZE 60 monitor
+CXI:SETRA:01:CH2_MASS 60 monitor
+CXI:SETRA:01:CH3_SIZE 60 monitor
+CXI:SETRA:01:CH3_MASS 60 monitor
+CXI:SETRA:01:CH4_SIZE 60 monitor
+CXI:SETRA:01:CH4_MASS 60 monitor
+CXI:SETRA:01:CH5_SIZE 60 monitor
+CXI:SETRA:01:CH5_MASS 60 monitor
+"""
```

Comment:
We should replace these notes with a more informative docstring for the module.  Maybe something like:
```
"""
Module to define classes for the Facility Monitoring System (FMS).
"""
```

Though more detail is always appreciated